db/readMe_DB.md

```diff
@@ -0,0 +1,27 @@
+#Entwicklung mit sqlite3 und db-migrate
+
+generell sind alle benötigten npm-Module in der backend/package.json hinterlegt
+nachfolgende Befehle sind also aus dem backend Ordner heraus auszuführen
+
+##Aktualisieren der Datenbank
+- wenn noch nicht geschehen:
+>npm i
+- um die DB auf den neusten Stand zu bringen (wenn die DB noch nicht existiert wird sie erstellt):
+>node_modules/db-migrate/bin/db-migrate up
+- um die DB um die letzte Migration zurückzurollen:
+>node_modules/db-migrate/bin/db-migrate down
+
+- Für weitere Informationen bitte hier nachlesen:
+  https://db-migrate.readthedocs.io/en/latest/Getting%20Started/commands/
+##Migrationen
+- migrationen liegen unter backend/migrations
+- eine neue Migration kann erstellt werden mit:
+>node_modules/db-migrate/bin/db-migrate create nameDerMigration
+- alternativ auch mit sql Dateien mit:
+>node_modules/db-migrate/bin/db-migrate create nameDerMigration --sql-file
```

Comment:
Soll man nach --sql-file Dateien in den Befehl als parameter übergeben oder wird man dann nach Dateien in einem Dialog gefragt?

Reply:
Die SQL Dateien werden dann automatisch generiert

Reply:
Hab den Text angepasst, sodass es hoffentlich deutlicher ist

Reply:
Ne der neue Text ist super verwirrend. Ich schlage vor:
"Es können auch direkt sql Dateien mit generiert werden:"

Reply:
Habs angepasst und noch ne kleine Erläuterung dazu geschrieben